### DISCUSSION TITLE
Despliegue de mi pagina html

### DISCUSSION THREAD

**Discussion by jmlimas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#314] [General]
Hola  amigos   les recomiendo este  sitio para  publicar gratis 

https://www.netlify.com/

esta es mi  publicación  que  me  recomiendan mejorar..  
https://quizzical-booth-0ced4f.netlify.app/

**Comment by JuanSalvador73** [comment]
Excelente. Es responsiva y además ya simulaste muy bien la funcionalidad. A mi ver está mas que completa.
